tests/playwright-test/playwright.ct-build.spec.ts

```diff
@@ -40,7 +40,7 @@ test('should work with the empty component list', async ({ runInlineTest }, test
   expect(metainfo.version).toEqual(require('playwright-core/package.json').version);
   expect(metainfo.viteVersion).toEqual(require('vite/package.json').version);
   expect(Object.entries(metainfo.tests)).toHaveLength(1);
-  expect(Object.entries(metainfo.sources)).toHaveLength(8);
+  expect(Object.entries(metainfo.sources)).toHaveLength(9);
```

Comment:
Due to usage of new `/client` entrypoint. Fixes https://github.com/microsoft/playwright/actions/runs/3817462646/jobs/6493769325#step:8:48